### PR TITLE
completed js loops - Mohamed 

### PR DESCRIPTION
**Pull request by Mohamed-Elmessiry**
Screen Shot 2021-03-17 at 2 02 11 PM

### CODE REVIEW

**Comment by Mohamed-Elmessiry** [comment]
sorry wrong screen shot 
Screen Shot 2021-03-17 at 10 07 57 PM

File: javascript-loops/main.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+  }
+  return numbers;
+}
+
+function getEvenNumbersToTwenty() {
+  var evenNumbers = [];
+  for (var i = 2; i <= 20; i += 2) {
+    evenNumbers.push(i);
+
+  }
```
Comment (codingcodymiller) [inline javascript-loops/main.js:25]: Please use a `while` loop for this

File: javascript-loops/main.js [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+  return evenNumbers;
+}
+
+function repeatWord(word, times) {
+  var count = 1;
+  var repeated = '';
+  for (var i = count; i < times; i++) {
+    repeated += word;
+    count++;
+  }
```
Comment (codingcodymiller) [inline javascript-loops/main.js:35]: Please use a `while` loop for this

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Nice work so far!! Please see my notes

File: javascript-loops/main.js [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+            getValues
+ */
+
+function getNumbersToTen() {
+  var numbers = [];
+  var currentNumber = 1;
+  for (var i = 0; i < 10; i++) {
+    numbers.push(currentNumber);
+    currentNumber++;
+  }
```
Comment (codingcodymiller) [inline javascript-loops/main.js:16]: This should be a `while` loop

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

File: javascript-loops/main.js (hunk 3 above)
Comment (Mohamed-Elmessiry) [reply javascript-loops/main.js:16]: Hi Cody, just to clarify when is it best to use a while loop ?

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**